public/app/plugins/datasource/influxdb/datasource.ts

```diff
@@ -554,18 +581,6 @@ export default class InfluxDatasource extends DataSourceWithBackend<InfluxQuery,
     });
   }
 
-  getTagKeys(options: any = {}) {
-    const queryBuilder = new InfluxQueryBuilder({ measurement: options.measurement || '', tags: [] }, this.database);
-    const query = queryBuilder.buildExploreQuery('TAG_KEYS');
-    return this.metricFindQuery(query, options);
-  }
-
-  getTagValues(options: any = {}) {
-    const queryBuilder = new InfluxQueryBuilder({ measurement: options.measurement || '', tags: [] }, this.database);
-    const query = queryBuilder.buildExploreQuery('TAG_VALUES', options.key);
-    return this.metricFindQuery(query, options);
-  }
-
```

Comment:
Those weren't being used so I decided to delete them. We have same functions in `public/app/plugins/datasource/influxdb/influxQLMetadataQuery.ts`

Reply:
I put them back https://github.com/grafana/grafana/pull/66466/commits/9f344bd7bbf59236385f8a8094dd9ab903ab17d3